### PR TITLE
Indentation

### PR DESCRIPTION
**Pull request by poohzrn**
Indented all `.vim` files and inserted `vim: ts=4 sw=4 et `where missing

### CODE REVIEW

**Comment by poohzrn** [comment]
changes made with:
`nvim --headless $(find . -name '*.vim') -c "argdo :normal :gg=G" -c "x"`

**Comment by blueyed** [comment]
Thanks.
I've planned to do this also, but it causes unnecessary conflicts with all my local stashes/branches (and pending PRs).
Therefore I think it's not really worth it currently.

Apart from that:

1. I'd prefer to use `sw=2`
2. I would remove all modelines.  IMHO they only add noise, and this should be configured for the project, e.g. by using an `.editorconfig` file.

File: autoload/neomake/makers/ft/d.vim [hunk 1]
```diff
@@ -45,15 +45,15 @@ function! s:DmdStyleMaker(args)
...
-        \ }
+                \ 'args': l:args,
+                \ 'errorformat':
+                \     '%f(%l\,%c): %trror: %m,' .
+                \     '%f(%l): %trror: %m,' .
+                \     '%f(%l\,%c): %tarning: %m,' .
+                \     '%f(%l): %tarning: %m,' .
+                \     '%f(%l\,%c): Deprecation: %m,' .
+                \     '%f(%l): Deprecation: %m,',
+                \ }
```
Comment (davidosomething) [inline autoload/neomake/makers/ft/d.vim]: I prefer this style, where there is inline spacing to show that the following string is the value of the above key

File: autoload/neomake/makers/ft/cuda.vim [hunk 2]
```diff
@@ -6,10 +6,10 @@ endfunction
...
-        \ 'errorformat':
-            \ '%f\(%l\): %trror: %m,'.
-            \ '%f\(%l\): %tarning: %m,'.
-            \ '%f\(%l\): %m',
-        \ }
+                \ 'exe': 'nvcc',
+                \ 'errorformat':
+                \ '%f\(%l\): %trror: %m,'.
+                \ '%f\(%l\): %tarning: %m,'.
+                \ '%f\(%l\): %m',
```
Comment (davidosomething) [inline autoload/neomake/makers/ft/cuda.vim]: imo this is ugly, needs inline spacing to distinguish key and value

**Comment by poohzrn** [comment]
1. I don't have any preferences regarding `sw=2` vs. `sw=4`
2. `.editorconfig` is a great idea, however, I don't know any ways to to that other than using a [plugin](http://editorconfig.org/)
    To enforce this some a step to the ci stuff should be added - or do you have any suggestions?

File: autoload/neomake/utils.vim [hunk 3]
```diff
@@ -332,7 +332,7 @@ function! neomake#utils#hook(event, context) abort
     if exists('#User#'.a:event)
         let g:neomake_hook_context = a:context
         call neomake#utils#DebugMessage('Calling User autocmd '.a:event
-                                      \ .' with context: '.string(a:context))
+                    \ .' with context: '.string(a:context))
```
Comment (davidosomething) [inline autoload/neomake/utils.vim]: this should have been caught by vint -- needs some spacing around the `.`

**Comment by davidosomething** [comment]
there are several plugins, dahu and sgur have vimL only implementations of editorconfig as well

for the sw, I prefer 2 also since most VimL as well as the helpfiles are formatted for tw of 80

**Comment by poohzrn** [comment]
I think we should just close the pull request and ping #597

File: autoload/neomake/utils.vim (hunk 3 above)
Comment (blueyed) [inline autoload/neomake/utils.vim]: @davidosomething 
It's only style, not an error?!  AFAIK vint does not handle this, does it?

**Comment by blueyed** [comment]
Ok.

Let's go with `sw=2` then.

What about the modelines?
https://github.com/embear/vim-localvimrc is also useful for project-wide settings.  Maybe we could add such a file, additionally to an editorconfig?
I don't use an editorconfig plugin myself - the last time I've looked it seemed like too much overhead.
@davidosomething 
Do you use one?  Can you recommend one?

> To enforce this some a step to the ci stuff should be added

Yes, would be nice.  What could we use here?

**Pull request closed without merging**